api/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +80,22 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Global target percentage of requests that will be force traced if the
+    // x-client-trace-id header is set. Must be an integer number between 0 and
+    // 100.
+    uint32 client_enabled = 3 [(validate.rules).uint32.lte = 100];
+
+    // Global target percentage of requests that will be traced after all other
+    // checks have been applied (force tracing, sampling, etc.). Must be a
+    // number between 0 and
+    // 100.
```

Comment:
Please specify the default as done in https://www.envoyproxy.io/docs/envoy/latest/configuration/http_conn_man/runtime.html?highlight=sample.

Reply:
Fixed.